cluster.go

```diff
@@ -1204,8 +1204,14 @@ func (c *Cluster) Version() string {
 
 // Peers returns the IDs of the members of this Cluster.
 func (c *Cluster) Peers(ctx context.Context) []api.ID {
-	ctx, span := trace.StartSpan(ctx, "cluster/Peers")
+	// To keep global shutdown semantics, create span from
```

Comment:
Shouldn't this happen in other places?

Reply:
I have now done this for all exported functions in cluster.go and a few special unexported ones.